### ISSUE TITLE
V2: UI Wireframe

### ISSUE THREAD

**Issue by roughpandaz** [nervosnetwork/docs.nervos.org#209]
## UI goals:

**User:** 
Phase 1: 1st-time devs - DApp builders
Phase 2: Contract devs

User needs:
1. Test (easy) - no setup test Dapp/Contract
2. Build (medium) - simple setup to build + test DApp
3. Customize (Hard) - custom dapps and contract interaction

**Initial Draft** ([figma](https://www.figma.com/file/oEwJriiUVv4tgLV5oU6tLx/Nervos-Docs---DRAFT?type=design&node-id=0-1&mode=design&t=0uJKbP1d1xFq8y8I-0))

## Key Pages:
- Getting Started
- Build: DApps
- Example Pages

Example pages should be split into the following:
- Summary: 2-3 lines on what devs will learn
- Setup: Dev Environment (1 line)
- Start: copy code and start project (use defaults and hardcoded address for local-chains)
- Understand: Learn about key parts of DApp.
- Customize: Customize and config project.
- Next: What should developers need to do after this.

Other considerations:
- [ ] Cookies + TOS: Accept (simple button)

## Menu
**Build: DApp**
Examples
Local Environment
Contracts --> Explain what contracts are doing
DevTools --> Link to dev tool packages
RPC --> useful tools for dev builders

**Build: Contract**
Examples
DevTools

**Ecosystem**
Explorer
TestNet Faucet

**Integrate**
Wallets
Other Applications

**Concepts**
Cells
...
...

**Mining**
Run a Node
...

**Help**
Discord
Email
Twitter

**Comment by yfeng2824** [comment]
I think the above menu should be re-organized for scalability and visibility. Thus, I came up with this categorization to be placed on the page header: 
![Screenshot 2024-02-22 at 4 53 16 PM](https://github.com/nervosnetwork/docs.nervos.org/assets/140578792/e77a02f0-c0d7-4060-8b9c-d88938645982)
---
Figma link: https://www.figma.com/file/WIrbDAZH7Cm1RyfDXaBLVE/Documentation?type=design&node-id=15%3A32&mode=design&t=OuWH2TY3eUnoQJKy-1
I included 3 pages:
1. Build Overview 
2. Hello CKB tutorial - detail page (users can run code via StackBlitz directly)
3. Create First Spore tutorial - detail page (users need to create project on their local env)
